shell/platform/android/io/flutter/embedding/android/FlutterActivity.java

```diff
@@ -488,10 +489,16 @@ private Drawable getSplashScreenFromManifest() {
       Bundle metaData = getMetaData();
       int splashScreenId = metaData != null ? metaData.getInt(SPLASH_SCREEN_META_DATA_KEY) : 0;
       return splashScreenId != 0
-          ? Build.VERSION.SDK_INT > Build.VERSION_CODES.LOLLIPOP
+          ? Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP
               ? getResources().getDrawable(splashScreenId, getTheme())
               : getResources().getDrawable(splashScreenId)
           : null;
+    } catch (Resources.NotFoundException e) {
+      Log.w(
+          TAG,
+          "Splash screen not found. Ensure the drawable exists and that it's valid."
+              + " This can also occur if the xml has unresolved theme references.");
+      return null;
```

Comment:
Developers are likely to miss this log right? Adding the log is good, but I think we should rethrow.

Reply:
+1 - Although Flutter run streams the logcat. However, logcat tends to be verbose and useless unless someone knows what to look at.

Reply:
I think the log is fine, because the tool will filter logcat for messages coming from the engine and this will pass that filter right? This also seems to be actionable by the developer. But yeah we should definitely add the linting.